.github/workflows/cifuzz.yml

```diff
@@ -0,0 +1,26 @@
+name: CIFuzz
+on: [pull_request]
+jobs:
+  Fuzzing:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Build Fuzzers
+      id: build
+      uses: google/oss-fuzz/infra/cifuzz/actions/build_fuzzers@master
+      with:
+        oss-fuzz-project-name: 'linkerd2-proxy'
+        dry-run: false
+        language: rust
+    - name: Run Fuzzers
+      uses: google/oss-fuzz/infra/cifuzz/actions/run_fuzzers@master
```

Comment:
You might run into a potential problem if you pin CIFuzz to a particular SHA.
To build your fuzzers, CIFuzz relies on your Dockerfile and build.sh in the OSS-Fuzz repo as well as the code in this repo.
If you update the Dockerfile or build.sh in OSS-Fuzz and then make a PR in this repo, CIFuzz will be using the stale versions of your files.
If you remember to update this SHA every time those files are updated, then there won't be an issue.
Otherwise, you can avoid ever being in this situation by making the build.sh file in OSS-Fuzz call some script in this repo.
Then, you will never need to modify the build.sh in OSS_Fuzz.